content/doc/developer/plugin-development/pipeline-integration.adoc

```diff
@@ -0,0 +1,13 @@
+---
+layout: developerguide
+title: Writing Pipeline-Compatible Plugins
+references:
+- url: https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md
+  title: Plugin Developer Guide
+- url: https://github.com/jenkinsci/pipeline-plugin/blob/master/COMPATIBILITY.md
+  title: Plugins Compatible with Pipeline
```

Comment:
Useless. Point to https://github.com/jenkinsci/workflow-step-api-plugin/blob/master/README.md instead.